website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -1837,14 +1837,15 @@ var Range = ace.require('ace/range').Range;
     };
 
     commandProto.addLinkDef = function (chunk, linkDef) {
-
+        

```

Comment:
(That'll introduce noise in the opposite direction for this particular PR, since the file in question already includes a number of lines with trailing whitespace. Such is hygiene?)